dev/com.ibm.ws.cdi.interfaces/src/io/openliberty/cdi/spi/CDIExtensionMetadata.java

```diff
@@ -68,6 +68,8 @@ public interface CDIExtensionMetadata {
      * All classes returned by this method will be will be found by CDI during type discovery
      * so that they can then be used as beans (or interceptors etc. if annotated as such) by the application.
      * All classes must be in the same archive as your CDIExtensionMetadata.
+     *
+     * If a bean registered via this does not have a bean defining annotation it will default to a scope of @Dependent
      */
```

Comment:
As this is SPI, the Javadoc is published so we should format it properly.
```suggestion
     * All classes must be in the same archive as your CDIExtensionMetadata.
     * <p>
     * If a bean registered via this does not have a bean defining annotation it will default to a scope of {@code @Dependent}
     */
```